core/src/main/java/io/substrait/expression/ExpressionCreator.java

```diff
@@ -90,21 +101,35 @@ public static Expression.TimestampTZLiteral timestampTZ(boolean nullable, long v
   }
 
   public static Expression.TimestampTZLiteral timestampTZ(boolean nullable, Instant value) {
-    var epochMicro = TimeUnit.SECONDS.toMicros(value.getEpochSecond()) + TimeUnit.NANOSECONDS.toMicros(value.getNano());
+    var epochMicro =
+        TimeUnit.SECONDS.toMicros(value.getEpochSecond())
+            + TimeUnit.NANOSECONDS.toMicros(value.getNano());
     return timestampTZ(nullable, epochMicro);
   }
 
-  public static Expression.IntervalYearLiteral intervalYear(boolean nullable, int years, int months) {
-    return Expression.IntervalYearLiteral.builder().nullable(nullable).years(years).months(months).build();
+  public static Expression.IntervalYearLiteral intervalYear(
+      boolean nullable, int years, int months) {
+    return Expression.IntervalYearLiteral.builder()
+        .nullable(nullable)
+        .years(years)
+        .months(months)
+        .build();
   }
 
   public static Expression.IntervalDayLiteral intervalDay(boolean nullable, int days, int seconds) {
-    return Expression.IntervalDayLiteral.builder().nullable(nullable).days(days).seconds(seconds).build();
+    return Expression.IntervalDayLiteral.builder()
```

Comment:
this is in fact not due to line length but rather being caused by the google-java-formatter. The formatter is particularly made for code readability and it splits statements like this one into multiple lines. 
Having the code being formatted like this makes diffs eventually easier to review, because a single change within this statement then only causes a diff that is immediately visible (because you don't have to go through the entire LOC to spot where the diff in this long statement is).

I didn't have time yet to update the PR and add some reasoning why this is using the google formatter, but will do so in a bit.

Reply:
> add some reasoning why...

I merged a couple other patches in the meantime. Hopefully won't be too disruptive.

Reply:
not a problem at all. I re-applied all changes and pushed them. Note however that the formatting changes to the gradle kotlin files were overwritten with https://github.com/substrait-io/substrait-java/commit/83abf918c8db778551c8b4b8f638db457ba04b28 so I re-applied those again as part of 
ff7fcf5